wails-frontend/src/pages/PasswordPromptHandler/OperationCost.tsx

```diff
@@ -0,0 +1,170 @@
+    return new BigNumber(coins).plus(new BigNumber(fees)).toFixed(9);
+  }
+
+  return (
+    <div className="w-full">
+      <div className="flex w-full justify-between pb-2">
+        <p>{Intl.t('password-prompt.sign.operation-const')}</p>
+        <p>
+          {formatStandard(operationCost)} {masToken}
+        </p>
+      </div>
+
+      <div className="flex flex-col w-full h-fit">
+        <AccordionCategory
+          isChild={false}
+          iconOpen={<FiChevronDown />}
+          iconClose={<FiChevronUp />}
+          customClass="px-0 py-0"
+          categoryTitle={
+            <p className="p-0 pb-2 mas-caption text-f-disabled-1">
+              {Intl.t('password-prompt.sign.view-details')}
+            </p>
+          }
+        >
+          <AccordionContent customClass="px-0 py-0">
+            <div className="flex flex-col gap-1 text-f-disabled-1">
+              <div className="flex flex-row justify-between">
+                <p className="flex flex-row mas-caption">
+                  <Tooltip
+                    className="mr-1"
+                    icon={<FiInfo size={16} />}
+                    content={
+                      <>
+                        {Intl.t('password-prompt.sign.fees-tooltip')}
+                        <br />
+                        {Intl.t('password-prompt.sign.fees-tooltip-2')}
+                      </>
+                    }
+                  />
+                  <label className="mas-caption">
+                    {Intl.t('password-prompt.sign.fees')}
+                  </label>
+                </p>
+                <InlineMoney
+                  customClass="mas-caption"
+                  disabled={!isEditing}
+                  value={fees}
+                  onValueChange={(event) => setFees(event.value)}
+                />
+              </div>
+              {!hideCoins && (
+                <div className="flex flex-row justify-between pb-2">
+                  <p className="flex flex-row mas-caption">
+                    <Tooltip
+                      className="mr-1"
+                      icon={<FiInfo size={16} />}
+                      content={
+                        <>
+                          {Intl.t('password-prompt.sign.coins-tooltip')}
+                          <br />
+                          {Intl.t('password-prompt.sign.coins-tooltip-2')}
+                        </>
+                      }
+                    />
+                    {Intl.t('password-prompt.sign.coins')}
+                  </p>
+                  <InlineMoney
+                    customClass="mas-caption"
+                    disabled
+                    value={formatStandard(coins)}
+                  />
+                </div>
+              )}
+              <div className="flex flex-row justify-end gap-1">
+                {isEditing ? (
+                  <>
+                    <button
+                      className="flex flex-row hover:cursor-pointer"
```

Comment:
tailwind tip: when using the tag "flex" attr "flex-row" is default, so you can safely remove  "flex-row"

Reply:
ok but there are many other flex flex-row in the code, maybe another PR to simplify all this

Reply:
Totally